airbyte-webapp/src/packages/cloud/App.tsx

```diff
@@ -38,7 +38,9 @@ const Services: React.FC = ({ children }) => (
       <NotificationServiceProvider>
         <ConfirmationModalService>
           <FormChangeTrackerService>
-            <FeatureService>
+            <FeatureService
+              features={[FeatureItem.AllowOAuthConnector, FeatureItem.AllowCreateConnection, FeatureItem.AllowSync]}
```

Comment:
I'd prefer to have it inline, given that we're not needing to import it anywhere else, it would just make the code more fragmented and requires you to follow into other files to see the feature set. I'd also have kept the OSS one inline if we wouldn't have needed to import it for the tests (and even considered just duplicate it into the tests).